tests/assets/move-projects/car-wash-example/sources/CarWash.move

```diff
@@ -0,0 +1,92 @@
+/// This module defines a minimal Coin and Balance.
+module DeveloperBob::CarWash {
+    use std::signer;
+    use substrate::balance;
+
+    /// Address of the owner of this module
+    const MODULE_OWNER: address = @DeveloperBob;
+
+    /// Simple solution, fixed price for one washing coin.
+    const COIN_PRICE: u128 = 1000000000000000000; // equals 1 UNIT
+
+    /// Error codes
+    const NOT_MODULE_OWNER: u64 = 0;
+    const MODULE_NOT_INITIALIZED: u64 = 1;
+    const MODULE_ALREADY_INITIALIZED: u64 = 2;
+    const USER_ALREADY_EXISTS: u64 = 3;
+    const USER_DOES_NOT_EXIST: u64 = 4;
+    const NO_COINS_AVAILABLE: u64 = 5;
+    const COIN_LIMIT_REACHED: u64 = 6;
+
+    /// Struct stores number of coins for each user.
+    struct Balance has key, store {
+        coins: u8
```

Comment:
Seems like Bob is going to handicap it's business 😃

Reply:
Yeah, I thought, if Bob would hand out 255 coins for his car wash, he would have another problem I think :thinking: :laughing: